app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js

```diff
@@ -48,10 +49,20 @@ class TSnePlotViewRoute extends React.Component {
   }
 
   render() {
+
+    console.log(this.props)
```

Comment:
You probably left this line accidentally in the code.
Could you remove it, please?
Thanks